installer/build.sbt

```diff
@@ -147,10 +147,10 @@ lazy val ossDockerfile: Def.Setting[sbt.Task[sbtdocker.DockerfileBase]] = {
         workDir(targetDir)
         runRaw("apk update && apk add wget bash curl")
         runRaw(
-          "wget -q https://github.com/openshift/origin/releases/download/v3.11.0/openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit.tar.gz -O - | tar -zxf - --strip-components=1 --exclude kubectl -C /usr/local/bin"
```

Comment:
Note that `strip-components` doesn't work in Alpine.

Reply:
Thanks!